Release notes for 20.05 (#9838)
galaxyproject/galaxy: doc/source/releases/20.05_announce_user.rst

===========================================================
May 2020 Galaxy Release (v 20.05)
===========================================================

.. include:: _header.rst

Highlights
===========================================================

**Mass-Spec datatypes**
  @bernt-matthias, can you write a bit about this?

**New Interactive tools**
  @astrovsky01, can you write this ?

> **Feature3**

Reviewer: I'd recommend the following highlights, maybe in this order:

1. Tool recommendations (even if not on every galaxy, can be, and it is a user requested/facing feature.)
2. New ITs
3. Fixed the rule builder scrolling (huge bug for many medium power users.), todo find xref.
3. OIDC Login "yay social auth, login with fav provider blah blah"
  - https://github.com/galaxyproject/galaxy/pull/9060
  - Connect/disconnect interface
  - https://github.com/galaxyproject/galaxy/pull/9162 logout
  - elixir aai saw improvements

ITs is good, mass-spec is maybe too specific for this announcement?

I'd recommend against startup speed + tool testing improvements, these are really end user announcements and should focus on things people who don't run or develop galaxy, people who just show up to .org/.eu

Author: Fair, I'll move / drop those.

New Visualizations
===========================================================

.. visualizations
* Update charts entrypoint inclusion of dist artifacts
  `Pull Request 9511`_
* Editor as visualization
  (thanks to `@OlegZharkov <https://github.com/OlegZharkov>`__).
  `Pull Request 9531`_
* Chira visualization for Chimeras
  (thanks to `@OlegZharkov <https://github.com/OlegZharkov>`__).
  `Pull Request 9562`_
* Skip visualization build, if no changes were registered
  (thanks to `@OlegZharkov <https://github.com/OlegZharkov>`__).
  `Pull Request 9634`_

New Datatypes
===========================================================

.. datatypes
* new data types for OpenMS
  (thanks to `@bernt-matthias <https://github.com/bernt-matthias>`__).
  `Pull Request 8109`_
* Add tpr datatype
  (thanks to `@simonbray <https://github.com/simonbray>`__).
  `Pull Request 9244`_
* Python3 compatible SnpEff and SnpSift metadata detection
  (thanks to `@wm75 <https://github.com/wm75>`__).
  `Pull Request 9282`_
* Add datatypes for xcms and camera
  (thanks to `@computational-metabolomics <https://github.com/computational-metabolomics>`__).
  `Pull Request 9470`_

> * index with -c flag for large bam (one region larger than 2**29 - 1)

Reviewer 2: ```suggestion
* index with ``-c`` flag for large bam (one region larger than ``2**29 - 1``)
```

Author: ```suggestion
* Create csi index for bam files where any region is larger than ``2**29 - 1``)
```

.. figure:: images/20.05-tool-prediction.gif
   :alt: Tool prediction shown in the workflow interface

**New Interactive Tools**
  Through the combined efforts of many UseGalaxy.eu admins and developers, as well as the European Galaxy community we have converted a significant number of interactive environments into the new interactive tool specification. Thanks to `@bgruening <https://github.com/bgruening>`__, `@hexylena <https://github.com/hexylena>`__, `@jenzopr <https://github.com/jenzopr>`__, `@yvanlebras <https://github.com/yvanlebras>`__, `@annefou <https://github.com/annefou>`__, `@abretaud <https://github.com/abretaud>`__, `@joachimwolff <https://github.com/joachimwolff>`__, and `@astrovsky01 <https://github.com/astrovsky01>`__. You can try them all on `live.usegalaxy.eu <https://live.usegalaxy.eu/>`__

.. figure:: images/20.05-new-its.png
   :alt: Tool prediction shown in the workflow interface

> **Rule Based Uploader: Scolling!**

Reviewer 3: ```suggestion
**Rule Based Uploader: Scrolling!**
```
